app/views/devise_ghost/registrations/new.html.erb

```diff
@@ -0,0 +1,47 @@
+<div class="auth-container">
+  <h1 class="header">Create your account</h1>
+  <% if alert.present? %>
+    <p class="alert"><%= alert %></p>
+  <% end %>
+  <%= form_for(resource, as: resource_name, url: registration_path(resource_name)) do |f| %>
+    <div class="auth-form --signup">
+      <%= image_tag "default-avatar.png", :class => "avatar-placeholder" %>
+      <div
+        class="form-group"
+      >
```

Comment:
I know that you are used to react.js but in Rails keep things like this in one line like `<div class="form-group">`